packages/lit-element/src/test/node-imports.ts

```diff
@@ -21,13 +21,26 @@ import 'lit-element/decorators/query-assigned-elements.js';
 import 'lit-element/decorators/query-assigned-nodes.js';
 import 'lit-element/decorators/query-async.js';
 
-import {LitElement, html} from 'lit-element';
-import {customElement} from 'lit-element/decorators.js';
+import {LitElement, html, css} from 'lit-element';
+import {customElement, property} from 'lit-element/decorators.js';
```

Comment:
I wonder if checking the styles and property here is necessary if we're doing it in the test for `ReactiveElement`. Though it doesn't hurt to check again.

Reply:
Done